include/muu/all.h

```diff
@@ -36,6 +36,7 @@
 #include "half.h"
 #include "hashing.h"
 #include "integer_literals.h"
+#include "integer.h"
```

Comment:
A more specific name for this like `integer_aliases.h` would fit better in alongside `integer_literals.h`.